civictechprojects/views.py

```diff
@@ -581,6 +581,40 @@ def recent_projects_list(request):
         return JsonResponse({'projects': hydrated_project_list})
 
 
+def limited_listings(request):
+    """Summarizes current positions in a format specified by the LinkedIn "Limited Listings" feature."""
+
+    def cdata(str):
+        # Using CDATA tags (and escaping the close sequence) protects us from XSS attacks when
+        # displaying user provided string values.
+        return f"<![CDATA[{str.replace(']]>', ']]]]><![CDATA[>')}]]>"
+    
+    def position_to_job(position):
+        project = position.position_project
+        roleTag = Tag.get_by_name(position.position_role.first().slug)
+
+        return f"""
+        <job>
+            <company>{cdata(project.project_name)}</company>
+            <title>{cdata(roleTag.display_name)}</title>
+            <description>{cdata(position.position_description)}</description>
+            <location>{cdata(project.project_location)}</location>
+            <applyUrl>{cdata(position.description_url)}</applyUrl>
```

Comment:
This position description url is sometimes blank, so we might fall back to the project profile url in that case.

Reply:
Sure, done.